pkg/shell/indexes.js

```diff
@@ -562,6 +562,7 @@ function MachinesIndex(index_options, machines, loader, mdialogs) {
 
             item = compiled.items[component_manifest];
             label = item ? item.label : "";
+            $(frame).attr('title', label);
```

Comment:
Does it really make sense to explicitly set an empty title, instead of having no title at all?

Reply:
I guess it does not, I'll update the logic.